Makefile

```diff
@@ -278,25 +282,38 @@ add-replaces-field: ## Add replaces field to the CSV
 			exit 1; \
 		else \
 		  	# preferring sed here, in order to have "replaces" near "version" \
-			sed -r -i "/  version: $(VERSION)/ a\  replaces: $(OPERATOR_NAME).v$(PREVIOUS_VERSION)" ${BUNDLE_CSV}; \
+			sed -r -i "/  version: $(VERSION)/ a\  replaces: $(OPERATOR_NAME).v$(PREVIOUS_VERSION)" ${CSV}; \
 		fi \
 	fi
 
 .PHONY: bundle-reset-date
 bundle-reset-date: ## Reset bundle's createdAt
-	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ${BUNDLE_CSV}
+	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ${CSV}
 
 .PHONY: bundle-community-k8s
 bundle-community-k8s: bundle-community ## Generate bundle manifests and metadata customized to Red Hat community release
 
-.PHONY: bundle-community-rh
-bundle-community-rh: bundle-community  ## Generate bundle manifests and metadata customized to Red Hat community release
-	echo -e "\n  # Annotations for OCP\n  com.redhat.openshift.versions: \"v${OCP_VERSION}\"" >> bundle/metadata/annotations.yaml
+.PHONY: bundle-community-okd
+bundle-community-okd: bundle-community  ## Generate bundle manifests and metadata customized to Red Hat community release
 	$(MAKE) add-replaces-field
+	$(MAKE) add-ocp-annotations
+	echo -e "\n  # Annotations for OCP\n  com.redhat.openshift.versions: \"v${OCP_VERSION}\"" >> bundle/metadata/annotations.yaml
+
+.PHONY: add-ocp-annotations
+add-ocp-annotations: yq ## Add OCP annotations
```

Comment:
Is there a reason to add it here (upstream) and not just downstream?

Reply:
OKD is also verifying their existence.

https://github.com/medik8s/self-node-remediation/pull/221

I will update the commit message